jira/client.py

```diff
@@ -3544,6 +3544,17 @@ def projectcategories(self):
 
         return data
 
+    @lru_cache(maxsize=None)
+    def labels(self):
+
+        url = self._options["server"] + "/rest/api/3/label"
```

Comment:
Would transition this to use `self._get_json()`

Reply:
added an example in https://github.com/pycontribs/jira/pull/917#discussion_r739844741